src/assets/urls.ts

```diff
@@ -5,7 +5,7 @@ export const registerUrl = `${baseUrl}/register`;
 export const abilityUrl = `${baseUrl}/ability`;
 export const levelUrl = `${abilityUrl}/levels`;
 export const providerUrl = `${baseUrl}`;
+export const requestUrl = `${baseUrl}/request/provider`;
```

Comment:
Ar fi mai ok daca requestUrl ar fi doar "${baseUrl}/request"
`${baseUrl}/request/provider` ii specific pentru requestul de get by provider
in cod dupa ar fi:  ${requestUrl}/provider/${id}

Reply:
Am refactorizat aici